configs/ignite.json

```diff
@@ -9,22 +9,42 @@
           "js": "let versions=[]; for (let b of document.querySelectorAll('.rst-other-versions dd')){versions.push(b.innerText)};return JSON.stringify(versions)"
         }
       }
+    },
+    {
+      "url": "https://pytorch-ignite.ai/",
+      "selectors_key": "pytorch-ignite-ai"
     }
```

Comment:
You will see results from both websites on both search instances with this setup. If you'd like to limit the search to its website, you can use [`tags`](https://docsearch.algolia.com/docs/config-file/#using-custom-tags) for each URL group, and then in your frontend have:

on https://pytorch.org/ignite/
```jsx
<DocSearch
  // ... your credentials
  searchParameters={{
    facetFilters: ['tags:ignite']
  }}
/>
```

and on https://pytorch-ignite.ai/
```jsx
<DocSearch
  // ... your credentials
  searchParameters={{
    facetFilters: ['tags:ignite-ai']
  }}
/>
```

Reply:
concrete example

config: https://gist.github.com/shortcuts/b4b4e5ae8219a5e2496eb5fab2d2e1f6
live demo: https://codesandbox.io/s/bold-kate-g35nd?file=/src/App.js